internal/test/scorecard/tests.go

```diff
@@ -76,6 +77,34 @@ func CryostatCRTest(bundle *apimanifests.Bundle, namespace string, openShiftCert
 	return r.TestResult
 }
 
+// CryostatMultiNamespaceTest checks that the operator installs ClusterCryostat in response to a ClusterCryostat CR
+func CryostatMultiNamespaceTest(bundle *apimanifests.Bundle, namespace string, openShiftCertManager bool) *scapiv1alpha3.TestResult {
+	r := newTestResources(CryostatMultiNamespaceTestName)
+	namespaces := []string{"other-scorecard-namespace"}
```

Comment:
Maybe we could use something like `namespace + "other"`? So when run with our Makefile it would be `cryostat-operator-scorecard-other`? Although this is unlikely to be an existing namespace on the cluster, having something with the `cryostat` prefix makes it even less likely.

Reply:
Other than this, the PR is good to go